src/System/Console/Hawk/Args/Option.hs

```diff
@@ -78,6 +87,9 @@ instance Option HawkOption where
   helpMsg RecordDelimiter            = ["default '\\n'"]
   helpMsg OutputFieldDelimiter       = ["default <field-delim>"]
   helpMsg OutputRecordDelimiter      = ["default <record-delim>"]
+  helpMsg InPlaceEdit                = ["overwrite <file>, optionally creating"
+                                       ,"a backup copy at <file><suffix>"]
```

Comment:
Now that you are adding a `.` between the file and the suffix, please update the comment to say `<file>.<suffix>`.